src/function.py

```diff
@@ -408,6 +458,18 @@ def create_request():
     return create_request
 
 
+def _set_console_logging_level():
```

Comment:
This and `_debug_logging_enabled` seem redundant, one really just calls the other. Can we  consolidate this into a single function?

Reply:
I personally like the abstraction it provides, as the calling function becomes more readable. But I have no strong opinion on this, no problem in changing it.

Reply:
I don't have a strong opinion here.

Reply:
I took a look at other methods and found a similar abstraction: `_send_log_entry()` being the only caller of `_infra_enabled()` or `_logging_enabled()`. If you don't mind, I'd like to keep this level of abstraction as it makes the resulting code more readable. Thanks a lot for your understanding :-)